src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -408,7 +405,9 @@ public void RemoveDataRepositories(IEnumerable<DataRepository> dataRepositories)
 
       private void removeColumns(IEnumerable<DataColumn> dataColumns)
       {
-         _dataBrowserPresenter.RemoveDataColumns(dataColumns);
+         var columnsToRemove = dataColumns.ToList();
+         _dataBrowserPresenter.RemoveDataColumns(columnsToRemove);
+         Chart.RemoveCurvesForColumns(columnsToRemove);
```

Comment:
Curves were not removed from the chart/ Thus the curves were staying and of course the curve settings as well.
There is also one setting that needs to be implemented in PKSim